packages/components/MemberList/components/InviteControl.js

```diff
@@ -38,12 +45,25 @@ const OPTIONS = Object.values(TEAM_ROLES).map(role => ({
 const InviteControl = ({
   className,
   member,
+  isNewMember = false,
   teamId,
   onClickAdd,
   onClickRemove,
   onChange,
-}) =>
-  member.teamIds && member.teamIds.includes(teamId) ? (
+}) => {
+  if (isNewMember) {
+    return (
+      <RoleSelector
+        name="role"
+        value={member.role}
+        options={OPTIONS}
+        className={className}
+        onChange={onChange}
+      />
+    );    
+  }
+  
+  return member.teamIds && member.teamIds.includes(teamId) ? (
```

Comment:
you may replace `member.teamIds && member.teamIds.includes(teamId)` with chaining
`member.teamIds?.includes(teamId)`
// nitpick

Reply:
Done